internal/brokers/examplebroker/examplebrokerbus.go

```diff
@@ -0,0 +1,107 @@
+package examplebroker
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/godbus/dbus/v5"
+	"github.com/godbus/dbus/v5/introspect"
+	"github.com/ubuntu/decorate"
+)
+
+const (
+	dbusObjectPath = "/com/ubuntu/auth/ExampleBroker"
+	dbusInterface  = "com.ubuntu.auth.ExampleBroker"
+)
+
+// Bus is the D-Bus object that will answer calls for the broker.
+type Bus struct {
+	broker *broker
+}
+
+// StartBus starts the D-Bus service and exports it on the system bus.
+func StartBus() (err error) {
+	defer decorate.OnError(&err, "could not start example broker bus")
+
+	conn, err := dbus.ConnectSystemBus()
```

Comment:
Just be aware this doesn’t work for tests. We are not going to test this one anyway, but think about it once you tackle the mock broker.

Reply:
Yeah, I imagined hahaha
Of course it wouldn't be that easy, right? :smiling_face_with_tear:

Reply:
you have a lot of examples in adsys, don’t despair :p